### PR TITLE
update item_ttt_nokia SPANISH translation

### PR DESCRIPTION
**Pull request by Tekiad**
@Tekiad from Steam workshop (NOKIA)

### CODE REVIEW

File: lua/terrortown/entities/items/item_ttt_nokia.lua [hunk 1]
```diff
@@ -79,29 +91,38 @@ else
...
+		local es_str = "El dispositivo más duro conocido por el hombre. \n\n"
 		
 		if hitgroupConvar:GetBool() then
 			en_str = en_str .. "When " .. (bulletConvar:GetBool() and "shot in the chest" or "damaged in the chest") .. ", blocks up to " .. durabilityConvar:GetInt() .. " damage."
 			cn_str = cn_str .. "上半身" ..(bulletConvar:GetBool() and "中弹" or "受伤") .."时抵挡共" .. durabilityConvar:GetInt() .. "伤害。"
+			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo en el pecho" or "o lastimado en este" .. ", bloquea hasta" .. durabilityConvar:GetInt() .. "de daño."
 		else
 			en_str = en_str .. "When " .. (bulletConvar:GetBool() and "shot" or "damaged") .. ", blocks up to " .. durabilityConvar:GetInt() .. " damage."
 			cn_str = cn_str .. (bulletConvar:GetBool() and "中弹" or "受伤") .. "时抵挡共" .. durabilityConvar:GetInt() .. "伤害。"
+			es_str = es_str .. "Cuando haya " .. (bulletConvar):GetBool( and "disparo" or "daño") .. ", bloquea hasta " .. durabilityConvar:GetInt() .. " de daño."
```
Comment (TheOnly8Z) [inline lua/terrortown/entities/items/item_ttt_nokia.lua]:
```suggestion
			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo" or "daño") .. ", bloquea hasta " .. durabilityConvar:GetInt() .. " de daño."

```

Comment (TheOnly8Z) [reply lua/terrortown/entities/items/item_ttt_nokia.lua]: GetBool() is a function called on the convar `bulletConVar`.

File: lua/terrortown/entities/items/item_ttt_nokia.lua [hunk 2]
```diff
@@ -34,6 +34,18 @@ if SERVER then
 		for _, v in ipairs(player.GetAll()) do
 			v:SetNWInt("NokiaDurability", -1)
 		end
+
```
Comment (TheOnly8Z) [inline lua/terrortown/entities/items/item_ttt_nokia.lua:37]: This snippet should be removed

File: lua/terrortown/entities/items/item_ttt_nokia.lua [hunk 3]
```diff
@@ -79,29 +91,38 @@ else
 		-- HOLY FUCK dynamic descriptions?! I am a literal god
 		local en_str = "The most durable electronic device known to man. \n\n"
 		local cn_str = "世界上最抗打的电子产品。\n\n"
+		local es_str = "El dispositivo más duro conocido por el hombre. \n\n"
 		
 		if hitgroupConvar:GetBool() then
 			en_str = en_str .. "When " .. (bulletConvar:GetBool() and "shot in the chest" or "damaged in the chest") .. ", blocks up to " .. durabilityConvar:GetInt() .. " damage."
 			cn_str = cn_str .. "上半身" ..(bulletConvar:GetBool() and "中弹" or "受伤") .."时抵挡共" .. durabilityConvar:GetInt() .. "伤害。"
+			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo en el pecho" or "o lastimado en este" .. ", bloquea hasta" .. durabilityConvar:GetInt() .. "de daño."
```
Comment (TheOnly8Z) [inline lua/terrortown/entities/items/item_ttt_nokia.lua]:
```suggestion
			es_str = es_str .. "Cuando haya " .. (bulletConvar:GetBool() and "disparo en el pecho" or "o lastimado en este") .. ", bloquea hasta" .. durabilityConvar:GetInt() .. "de daño."

```

Close all parenthesizes that were open. This pair is intended to cover the part concatenated into the string.

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by Tekiad** [comment]
I think I've added your changes. Please, check again as I'm not familiarized with your code 😄

**Pull request merged**